resources/views/platform/components/game/devbox-claim-management.blade.php

```diff
@@ -158,18 +175,29 @@ function completeClaim() {
         </form>
     @endif
 
-    <form
-        class="mb-1"
-        action="/request/set-claim/drop-claim.php"
-        method="post"
-        onsubmit="return dropClaim()"
-    >
-        {!! csrf_field() !!}
-        <input type="hidden" name="game" value="{{ $gameId }}">
-        <input type="hidden" name="claim_type" value="{{ $claimType }}">
-        <input type="hidden" name="set_type" value="{{ $claimSetType }}">
-        <button class="btn">Drop {{ ClaimType::toString($claimType) }} Claim</button>
-    </form>
+    @if ($primaryClaimStatus === ClaimStatus::InReview && $primaryClaimUser === $user)
+        <div class="ml-2">Cannot modify Claim in Review status</div>
```

Comment:
I wonder if there is alternative language we should use for this. Technically, the Junior Dev can still extend the claim (and will likely have to if it remains in the queue for some time), which is a subtle modification of the claim.

Reply:
The Extend Claim button is still available when applicable.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/f6d89701-5221-4b80-9272-e74eee3afaa3)

The functionality only prevents dropping the claim. Maybe it should say that? "Cannot Drop Claim while being reviewed".

Reply:
Yes, "Cannot Drop Claim while being reviewed" or "Cannot Drop Claim while in Review status" both make more sense.

Reply:
Went with "Cannot Drop Claim while In Review"